browser/extensions/api/brave_rewards_api.cc

```diff
@@ -1187,9 +1177,9 @@ BraveRewardsGetAnonWalletStatusFunction::Run() {
     return RespondNow(Error("Rewards service is not initialized"));
   }
 
-  rewards_service->GetAnonWalletStatus(base::Bind(
-        &BraveRewardsGetAnonWalletStatusFunction::OnGetAnonWalletStatus,
-        this));
+  rewards_service->GetAnonWalletStatus(base::BindOnce(
```

Comment:
@mariospr this is not valid as is, just pass `this` as it was before. `Bind` will accept a refcounted pointer

Reply:
Oops! true... fixed now, sorry for the second overlook